### ISSUE TITLE
Generator NEEDS to warn & ask before deleting files in the targeted directory

### ISSUE THREAD

**Issue by jskrzypek** [mean-expert-official/loopback-sdk-builder#103]
I lost a bunch of work just now because I mis-typed the directory. Hopefully I'll be able to recover the deleted files, but this is ***very*** irresponsible and user-unfriendly. There's also no mention in the wiki of this behavior.

**Comment by jonathan-casarrubias** [comment]
@jskrzypek I'm so sorry about this issue, I'm really ashamed that after 15k installations I just found the critical of this behaviour because of it happen to you.

There actually is a reference in the [WIKI](https://github.com/mean-expert-official/loopback-sdk-builder/wiki/6.-Extending-Client-Models) that specifies this behaviour, but I will be redundant about this on the Installation Section.

Previously all the documentation was in 1 mark down document, when I splitted the documentation into sections within the wiki I did not realize that this information was too far away from the installation section, which actually could led you to this situation.

I will be adding this information within the Installation section and I will also add a validation and confirmation within the generator ASAP.

Regards,
Jon
